job-dsls/jobs/compile_downstream_build.groovy

```diff
@@ -0,0 +1,206 @@
+/**
+ * Creates downstream pullrequest (PR) jobs for kiegroup GitHub org. units.
+ * These jobs execute the downstream build skipping tests and skipping the gwt compilation
+ * for a specific PR to make sure the changes do not break the downstream repos.
+ */
+import org.kie.jenkins.jobdsl.Constants
+
+def final DEFAULTS = [
+        ghOrgUnit              : Constants.GITHUB_ORG_UNIT,
+        branch                 : Constants.BRANCH,
+        timeoutMins            : 600,
+        label                  : "kie-rhel7 && kie-mem8g",
+        ghAuthTokenId          : Constants.GITHUB_AUTH_TOKEN,
+        upstreamMvnArgs        : "-B -e -T1C -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true -Denforcer.skip=true -Dcheckstyle.skip=true -Dfindbugs.skip=true -Drevapi.skip=true clean install",
+        downstreamMvnGoals     : "-B -e -nsu -fae clean install -Dfull=true -DskipTests -Dgwt.compiler.skip=true -Dgwt.skipCompilation=true",
+        artifactsToArchive     : [
+                "**/target/*.log",
+                "**/target/testStatusListener*",
+                "**/target/screenshots/**",
+                "**/target/kie-wb*wildfly*.war",
+                "**/target/kie-wb*eap*.war",
+                "**/target/kie-drools-wb*wildfly*.war",
+                "**/target/kie-drools-wb*eap*.war",
+                "**/target/kie-server-*ee6.war",
```

Comment:
@mbiarnes you told me that kie-server ee6 is no longer being generated, so it should probably be removed from this "what to archive" section.